packages/ui/src/app/pages/Profile/MyAccounts.tsx

```diff
@@ -57,7 +57,21 @@ export const MyAccounts = () => {
                 }
                 value={locked}
               />
-              <TokenValueStat title="Total recoverable" tooltipText="Lorem ipsum..." value={recoverable} />
+              <TokenValueStat
+                title="Total recoverable"
+                tooltipText={
+                  <>
+                    <TooltipText>
```

Comment:
The string from this param is passed to the `Tooltip` itself, so it's wrapped with the' TooltipText' component in the end. Can't we just pass a plain string here as a prop instead of the element?